azblob/zt_url_service_test.go

```diff
@@ -206,6 +206,78 @@ func (s *aztestsSuite) TestAccountListContainersMaxResultsSufficient(c *chk.C) {
 	c.Assert(len(response.ContainerItems) >= 2, chk.Equals, true)
 }
 
+func (s *aztestsSuite) TestPermanentDeleteAndUndelete(c *chk.C) {
+	c.Skip("Versioning on storage account must be disabled in order for soft delete to function.")
```

Comment:
Instead of skipping, try using secondary account to run the test cases.
https://github.com/Azure/azure-storage-blob-go/blob/da3335192ee9609890f4172c43d0750c65941b6f/azblob/zt_test.go#L340

```go
func getAlternateBSU() (ServiceURL, error) {
	return getGenericBSU("SECONDARY_")
}

func getPremiumBSU() (ServiceURL, error) {
	return getGenericBSU("PREMIUM_")
}

func getBlobStorageBSU() (ServiceURL, error) {
	return getGenericBSU("BLOB_STORAGE_")
}
```

Reply:
Tests are important for this particular scenario.